astro/src/components/docs/reference/_configuration-properties.mdx

```diff
@@ -274,9 +274,10 @@ export const addlJavaArgsText = 'Any additional arguments that you want to pass
     The default search index `refresh_interval`. Bulk operations (bulk user import and reindex) may temporarily relax this value for better performance and then set it back to this value. This value specifies the delay before updated User and Entity data are available to search requests. Must be a valid [Elasticsearch Date/Time Interval Value](https://www.elastic.co/guide/en/elasticsearch/reference/current/sql-functions-datetime.html), however setting this variable is not recommended.
   </APIField>
 
-  <APIField name="fusionauth-app.silent-mode" type="Boolean" defaults="true" since="1.19.0">
+  <APIField name="fusionauth-app.silent-mode" type="Boolean" since="1.19.0">
     Determines if FusionAuth uses [Silent Mode](/docs/get-started/download-and-install/silent-mode) during the startup process.
-    
+
+    Silent mode defaults to `true` if `database.url`, `database.username`, and `database.root.username` are all set.
```

Comment:
Does it not require database passwords to be set?

Reply:
Do we _override_ it to `true` in this case? Or only set `silent-mode` if it was omitted from the configuration?